app/views/mailing_list/steps/completed/shared/_book_a_callback.html.erb

```diff
@@ -7,17 +7,19 @@
 
 <div class="mailing-list__actions">
   <% if callback_available? %>
-    <%= link_to("Book a callback", callbacks_steps_path, class: "button") %>
-    <div>
-      <small>or call us now:</small>
-      <p class="telephone">0800 389 2500</p>
+    <p>
+      Call us on <a href="tel:08003892500">0800 389 2500</a>. Monday to Friday between 8:30am and 5:30pm, except on <a href="https://www.gov.uk/bank-holidays">bank holidays</a>.
```

Comment:
Not sure that the second sentence makes sense as a standalone now - not a complete sentence @philspray ?